main.go

```diff
@@ -32,6 +41,11 @@ func main() {
 			inputAsInt[i][sid] = value
 		}
 
+		if *fieldsParameter != "" && *charParameter != "" {
+			fmt.Println("cut: only one type of list may be specified.")
```

Comment:
Where are you printing the error message?

Reply:
in line 45 obvly

Reply:
_Sigh_. Where are you printing the error message **to**?

Reply:
to the terminal

Reply:
Please open fmt.Println() and answer a bit more precise.

Reply:
it writes to standard output

Reply:
And where do you usually write errors to?